utils/utils.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from framework import config
+from framework.ssh_connection_manager import SSHConnectionManager
+from noobaa_sa.defaults import HEALTH
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+conf_root = config.ENV_DATA["config_root"]
+health = HEALTH
+base_cmd = f"sudo /usr/local/noobaa-core/bin/node {health}"
+unwanted_log = "2>/dev/null"
```

Comment:
1. Please remove config_root from the global scope and set it as the default value for the config_root argument of the `get_noobaa_health_status` util function:
```
get_noobaa_health_status(config_root=config.ENV_DATA["config_root"], **kwargs)
```

Note that once this is done you should also remove lines 42 and 43 that try to assign a default value to the config_root argument:
```
    if config_root is None:
        config_root = conf_root
```

2. Since you're only using `health` to build `base_cmd`, please omit the global variable and just use `HEALTH` instead:
```
base_cmd = f"sudo /usr/local/noobaa-core/bin/node {HEALTH}"
```

3. Please move `base_cmd` from the global scope to the scope of the `get_noobaa_health_status` util function.

4. Since we also use `unwanted_log` in other files in the project, please consider making this a constant in `constants.py`.

Reply:
Fixed